src/pages/index.js

```diff
@@ -260,18 +256,19 @@ const newCard = new PopupWithForm({
         // "res" in the renderCard parameter takes all the data from the .then("res") and pushes all of the necessary info that the server has to the cards
         const card = renderCard(res);
         sectionRenderer.addItem(card);
-        newCard.close();
+        newCardPopup.close();
       })
       .catch((err) => {
         console.error("an error has occurred", err);
       })
       .finally(() => {
+        revertSaveButtonListener();
```

Comment:
* When I submit a new card the button text doesn't get its initial text back in `finally` http://joxi.ru/MAj5QOltYGdnD2  . And the avatar button as well